### ISSUE TITLE
Error when trying to scroll pane with transformed-glyphs

### ISSUE THREAD

**Issue by nsrahmad** [McCLIM/McCLIM#733]
Can be reproduced in `Drawing-text -> Text -> Rotation`. Trying to scroll in the drawing pane gives:
```lisp
The value
  58720751
is not of type
  (INTEGER 0 0)
when binding #:G2
   [Condition of type TYPE-ERROR]

Restarts:
 0: [ABORT] Return to application command loop.
 1: [ABORT] Return to application command loop.
 2: [RETRY] Retry SLIME REPL evaluation request.
 3: [*ABORT] Return to SLIME's top level.
 4: [ABORT] abort thread (#<THREAD "repl-thread" RUNNING {1005A39E03}>)

Backtrace:
  0: (XLIB:GCONTEXT-CLIP-MASK #<XLIB:GCONTEXT :0 58720624>)
  1: (MCCLIM-TRUETYPE::%RENDER-TRANSFORMED-GLYPHS #<MCCLIM-TRUETYPE::CLX-TRUETYPE-FONT  size=50 ascent=47525/1024 descent=12075/1024 units->pixels=25/1024> "Ciao" 144.0 267.30957 #<CLIM-INTERNALS::STANDARD..
  2: ((FLET SB-THREAD::WITH-MUTEX-THUNK :IN CLIM:MEDIUM-DRAW-TEXT*))
  3: ((FLET "WITHOUT-INTERRUPTS-BODY-1" :IN SB-THREAD::CALL-WITH-MUTEX))
  4: (SB-THREAD::CALL-WITH-MUTEX #<CLOSURE (FLET SB-THREAD::WITH-MUTEX-THUNK :IN CLIM:MEDIUM-DRAW-TEXT*) {7FFB2BDA1F1B}> #<SB-THREAD:MUTEX "draw-font" owner: #<SB-THREAD:THREAD "repl-thread" RUNNING {1005A..
  5: ((:METHOD CLIM:MEDIUM-DRAW-TEXT* (CLIM-CLX:CLX-RENDER-MEDIUM T T T T T T T T T T)) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> "Ciao" 200 250 0 NIL :CENTER :CENTER #<unused argument> #<unused argument>..
  6: ((FLET CALL-NEXT-METHOD :IN "/home/nsrahmad/common-lisp/McCLIM/Backends/common/medium.lisp"))
  7: ((:METHOD CLIM:MEDIUM-DRAW-TEXT* :AROUND (CLIM-BACKEND:MULTILINE-TEXT-MEDIUM-MIXIN T T T T T T T T T T)) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> "Ciao" 200 250 0 NIL :CENTER :CENTER 201 250 T) [fas..
  8: ((LAMBDA (SB-PCL::.ARG0. SB-PCL::.ARG1. SB-PCL::.ARG2. SB-PCL::.ARG3. SB-PCL::.ARG4. SB-PCL::.ARG5. SB-PCL::.ARG6. SB-PCL::.ARG7. SB-PCL::.ARG8. SB-PCL::.ARG9. SB-PCL::.ARG10.)) #<CLIM-CLX:CLX-RENDER-..
  9: ((:METHOD CLIM:REPLAY-OUTPUT-RECORD (CLIM-INTERNALS::DRAW-TEXT-OUTPUT-RECORD T)) #<CLIM-INTERNALS::DRAW-TEXT-OUTPUT-RECORD X 144.22533932822208d0:255.77466067178008d0 Y 212.35540868311875d0:287.644591..
 10: ((FLET CALL-NEXT-METHOD :IN "/home/nsrahmad/common-lisp/McCLIM/Core/clim-basic/recording.lisp"))
 11: ((FLET "GCONTINUATION36" :IN CLIM:REPLAY-OUTPUT-RECORD) #<unused argument>)
 12: ((FLET CLIM-INTERNALS::GRAPHICS-OP :IN CLIM:INVOKE-WITH-DRAWING-OPTIONS) #<unused argument>)
 13: ((:METHOD CLIM-INTERNALS::DO-GRAPHICS-WITH-OPTIONS-INTERNAL (CLIM:MEDIUM T T)) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> #<CLOSURE (FLET CLIM-INTERNALS::GRA..
 14: ((SB-PCL::EMF CLIM-INTERNALS::DO-GRAPHICS-WITH-OPTIONS-INTERNAL) #<unused argument> #<unused argument> #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> #<CLOSURE (..
 15: ((:METHOD CLIM-INTERNALS::DO-GRAPHICS-WITH-OPTIONS (CLIM:MEDIUM T)) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}> #<CLOSURE (FLET CLIM-INTERNALS::GRAPHICS-OP :IN CLIM:INVOKE-WITH-DRAWING-OPTIONS) {7FFB2B..
 16: ((LABELS #:G5 :IN CLIM:INVOKE-WITH-DRAWING-OPTIONS) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C3F3DA3}>)
 17: ((:METHOD CLIM-INTERNALS::INVOKE-WITH-SHEET-MEDIUM-BOUND (T NULL CLIM:PERMANENT-MEDIUM-SHEET-OUTPUT-MIXIN)) #<CLOSURE (LABELS #:G5 :IN CLIM:INVOKE-WITH-DRAWING-OPTIONS) {7FFB2BDA31BB}> #<unused argume..
 18: ((:METHOD CLIM:INVOKE-WITH-DRAWING-OPTIONS (CLIM:SHEET T)) #<CLIM-CLX::CLX-839-DRAWING-APP-PANE CLIM-DEMO::BACKEND-OUTPUT {100C283853}> #<CLOSURE (FLET "GCONTINUATION36" :IN CLIM:REPLAY-OUTPUT-RECORD)..
 19: ((SB-PCL::EMF CLIM:INVOKE-WITH-DRAWING-OPTIONS) #<unused argument> #<unused argument> #<CLIM-CLX::CLX-839-DRAWING-APP-PANE CLIM-DEMO::BACKEND-OUTPUT {100C283853}> #<CLOSURE (FLET "GCONTINUATION36" :IN..
 --more--
```

**Comment by dkochmanski** [comment]
I think I've fixed this issue some time ago. Are you certain your clx version has this commit: https://github.com/sharplispers/clx/commit/1f2b6f3fd83880c073f37b06bbfcf2f70e29527c?

**Comment by nsrahmad** [comment]
Now I remember that. I had the CLX master branch but decided to delete and get the quicklisp version. 

And I am glad I did that.

I have the latest March distribution of Quicklisp, but `clx` still has version `clx-20181210-git` (just before the above commit). Quicklisp always put the release note with `updated-project` section on their blog at http://blog.quicklisp.org/ and I searched their for `clx`. It looks like last time Quicklisp updated clx was in December 2018. Their are several CLX commits after that and 3 Quicklisp releases. But somehow Quicklisp failed to update CLX.
